### PR TITLE
Add hyperterm-material theme

### PR DESCRIPTION
**Pull request by eMaj-Fr**
Hyperterm Material : A material design inspired theme plugin for HyperTerm.
https://github.com/dperrera/hyperterm-material

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ Like `awesome-hyperterm`? Reach out to me and say *hi* on [Twitter](https://twit
 * [hyperseti](https://www.npmjs.com/package/hyperseti) - Dark - It's a theme based on [Seti](https://github.com/jesseweed/seti-ui), a subtle dark colored UI theme for Atom.
 * [hyperterm-oceanic-next](https://github.com/kasperlewau/hyperterm-oceanic-next) - Dark blue theme optimised for ES2015. Ported from [Oceanic Next](https://github.com/voronianski/oceanic-next-color-scheme).
 * [hyperterm-spacegray](https://www.npmjs.com/package/hyperterm-spacegray) – Port of the popular Spacegray theme, optimized for terminal usage.
-* [hyperterm-material](https://github.com/dperrera/hyperterm-material) - Dark - A material design inspired theme.
+* [hyperterm-material](https://www.npmjs.com/package/hyperterm-material) - Dark - A material design inspired theme.
```
Comment (bnb) [inline README.md]: What about the theme is inspired by Material Design? Does it differ at all from "the" Material Design?

Comment (eMaj-Fr) [inline README.md]: I have taken the description provided by the author. If you prefer "the", no problem, I can change it.

Comment (bnb) [inline README.md]: We definitely don't have to stick to the descriptions provided by the author. 😄 

Is there anything that you find aesthetically pleasing, or another reason why this theme is awesome? Would love to hear _your_ thoughts on it!

**Comment by matheuss** [comment]
Hey @eMaj-Fr, could you please create a comment here with [this checklist](https://github.com/bnb/awesome-hyperterm/blob/master/.github/PULL_REQUEST_TEMPLATE.md)? Thanks!

**Comment by eMaj-Fr** [comment]
## Checklist:



- [x] The title for my package or theme uses its npm title (hyperterm-plugin-example)
- [x] The link for my package or theme uses the npmjs.com link (https://npmjs.com/package/hyperterm-plugin-example)
- [x] There is a visual representation of what my plugin does in the repo. (Plugin: screenshot or gif || Theme: screenshot of the colors within HyperTerm)
- [x] **VERY IMPORTANT:** I've written a short (one sentence) description for my package or theme of why it's awesome and deserves to be on the list.

**Comment by eMaj-Fr** [comment]
- [x] The title for my package or theme uses its npm title (hyperterm-plugin-example)
- [x] The link for my package or theme uses the npmjs.com link (https://npmjs.com/package/hyperterm-plugin-example)
- [x] There is a visual representation of what my plugin does in the repo. (Plugin: screenshot or gif || Theme: screenshot of the colors within HyperTerm)
- [x] **VERY IMPORTANT:** I've written a short (one sentence) description for my package or theme of why it's awesome and deserves to be on the list.

**Comment by eMaj-Fr** [comment]
I don't understand the error in the build. Someone can help me ?

**Comment by matheuss** [comment]
Oh, relax man 😅 The error is not related to your link :) Sorry for the delay and thanks for the contribution! Merging now!

**Pull request merged**